README.md

```diff
@@ -1,2 +1,28 @@
+# WARNING
+
+This is WIP repo and it is under development. Use it at your own risk. 
+If you have use-cases for such thing to be developed, please submit
+an issue or PR with description of your needs / fixes.
+
 # meta-trenchboot-demo
+
 Meta layer for Trenchboot related demos
+
+
+# Building
+
+```
+SHELL=bash kas-docker --ssh-dir ~/ssh-keys build meta-trenchboot-demo/kas.yml
+```
+
+Change directory to build/tmp/deploy/images/pcengines-apu2 and flash the drive
+with bmap-tools:
```

Comment:
@Asiderr this sound like assumption that we compile on apu2, which would be unusual.

Reply:
@pietrushnic I precised that in the last commit.